public/app/features/users/TokenRevokedModal.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button, Modal } from '@grafana/ui';
+import config from 'app/core/config';
+
+export const TokenRevokedModal = () => {
+  const redirectToLogin = () => {
+    window.location.href = config.appSubUrl + '/';
+  };
+
+  return (
+    <Modal title="Token revoked" isOpen={true} onDismiss={redirectToLogin} onClickBackdrop={() => {}}>
+      <p>Your token has been revoked because you have reached the maximum amount of concurrent sessions.</p>
+      <p>Please, log in again.</p>
```

Comment:
```suggestion
      <p>Sign in again.</p>
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please